design/sketch.webidl

```diff
@@ -401,13 +401,20 @@ dictionary WebGPUShaderModuleDescriptor {
 interface WebGPUShaderModule {
 };
 
-// Description of the framebuffer attachments
-dictionary WebGPUAttachmentsStateDescriptor {
-    sequence<WebGPUTextureFormatEnum> formats;
-    // TODO other stuff like sample count etc.
+// Description of a single attachment
+dictionary WebGPUAttachment {
+    // Attachment data format
+    WebGPUTextureFormatEnum format;
+    // Number of MSAA samples
+    u32 samples;
 };
 
-interface WebGPUAttachmentsState {
+// Description of the framebuffer attachments
+dictionary WebGPUAttachmentsState {
```

Comment:
Why make it a dictionary?

Reply:
The point of this PR is to remove the attachment state as a handle, because there is no backend-specific data to associate with it. The render pass details is derived from the actual data passed to render pipeline creation and the beginning of a render pass.